batch/batch/aioclient.py

```diff
@@ -147,8 +154,8 @@ def __init__(self, client, id, attributes):
 
         j = await self._client._post('/jobs/create', json=doc)
 
-        return Job(self._client,
-                   j['id'],
+        return Job(self,
+                   j['job_id'],
```

Comment:
Does this need to be `Job(self, batch, j['job_id']...`

Reply:
No, I think this is fine. `self` is a Batch object. Should match the new arguments for constructing a job.

Reply:
Ah, right